### PR TITLE
Table requeriment sub

### PR DESCRIPTION
**Pull request by melvinjim**

### CODE REVIEW

File: app/models/requeriment_sub.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package models
+
+import (
+	"time"
+
+	"github.com/gofrs/uuid"
+)
+
+type Requirements_sub_types struct {
```
Comment (BryanMoslo) [inline app/models/requeriment_sub.go]: @mperezlamadrid Que ni Rob Pike ni Mark Bates vean esto.

Comment (BryanMoslo) [reply app/models/requeriment_sub.go]: This should be named `RequirementsSubTypes`. In Go, structs are named in PascalCase

File: migrations/20220727152706_create_table_requirement_types.up.fizz [hunk 2]
```diff
@@ -1,4 +1,4 @@
-    create_table("requirement_types") {
+create_table("requirement_types") {
     t.Column("id", "uuid", {primary: true})
     t.Column("department_id", "uuid", {})
     t.Column("name", "string", {})
```
Comment (BryanMoslo) [inline migrations/20220727152706_create_table_requirement_types.up.fizz:4]: Can you please add CreatedAt and UpdatedAt, you can use `t.Timestamps()`

File: migrations/20220727195202_create_table_requeriment_subs.up.fizz [hunk 3]
```diff
@@ -0,0 +1,6 @@
+create_table("requeriment_sub_types") {
+    t.Column("id", "uuid", {primary: true})
+    t.Column("requirement_type_id", "uuid", {})
+    t.Column("name", "string", {})
+    t.ForeignKey("requirement_type_id", {"requirement_types": ["id"]}, {"on_delete": "cascade"})
```
Comment (BryanMoslo) [inline migrations/20220727195202_create_table_requeriment_subs.up.fizz]: Can you please add CreatedAt and UpdatedAt, you can use `t.Timestamps()`

File: app/models/requeriment_sub.go [hunk 4]
```diff
@@ -0,0 +1,15 @@
+package models
+
+import (
+	"time"
+
+	"github.com/gofrs/uuid"
+)
+
+type RequerimentsSubTypes struct {
```
Comment (rdiazrovira) [inline app/models/requeriment_sub.go]: A typo here. The name should be "RequirementSubType". Also of the typo, this should be named in singular. @melvinjim

File: migrations/20220727195202_create_table_requeriment_subs.down.fizz [hunk 5]
```diff
@@ -0,0 +1 @@
+drop_table("requeriment_sub_types")
```
Comment (rdiazrovira) [inline migrations/20220727195202_create_table_requeriment_subs.down.fizz]: Same here. Please fix the typo.

**Review by rdiazrovira — CHANGES_REQUESTED** [review]
We have a couple of migrations files that are empty here. Please remove them If we are not going to use them.

File: app/models/requeriment_sub.go [hunk 6]
```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/gofrs/uuid"
 )
 
-type RequerimentsSubTypes struct {
+type RequirementSubTypes struct {
```
Comment (rdiazrovira) [inline app/models/requeriment_sub.go]: This represents an only sub type, so the name should be in singular. @melvinjim

Comment (rdiazrovira) [reply app/models/requeriment_sub.go]: Also, fix the file names where you see the same typos.

**Review by rdiazrovira — APPROVED** [review]

**Pull request merged**